nucypher/blockchain/eth/policies.py

```diff
@@ -112,54 +119,30 @@ def __find_ursulas(self, ether_addresses: List[str], target_quantity: int, timeo
 
             # Select an ether_address: Prefer the selection pool, then unknowns queue
             if ether_addresses:
-                ether_address = ether_addresses.pop()
+                ether_address = bytes(ether_addresses.pop(), encoding="ascii")
```

Comment:
This is 42 bytes? The `eth_utils` module includes functions to cast to/from bytes while preserving a valid checksum address.

Reply:
Yeah, well, as you can see, it unhexlify'd it elsewhere.

But more to the point: "preserving"?  As far as I can tell, it just checksums it again.  Is there such a thing as "preserving"?  Obviously it can't be with a hex encoding, if so.  The [document in question](https://github.com/ethereum/eth-utils#to_checksum_addressvalue---text) appears to re-checksum them, with a deterministic result per EIP-55.  What is this "preserving"?  Can you link me to the document you're talking about?

Reply:
To be clear: the unhexlify routine, which I have done below, is the obvious way forward (ie, `public_address`).  This change is essentially temporary to get this PR in.  Consider it similarly transitory to BrokenBob.  :-)